test/unit/helpers/http_api.js

```diff
@@ -169,8 +169,9 @@ describe('httpApi', () => {
 				done();
 			});
 
-			it('should call logger.log with string "GET req/url from 127.0.0.1"', done => {
-				expect(loggerMock.log.calledWith('GET req/url from 127.0.0.1'));
+			it('should call logger.log with string "GET api/url from 127.0.0.1"', done => {
+				expect(loggerMock.log.calledWith('GET api/url from 127.0.0.1')).to.be
+					.true;
```

Comment:
`calledWith` doesn't have to be checked with true

Reply:
Without `to.be.true` it is a false positive. Why do you say it doesn't have to be checked? Can you share the info about it?

Reply:
ah sorry what i mean is
`expect(loggerMock.log).to.be.calledWith('GET api/url from 127.0.0.1');`